frontend/src/components/AppFooter.vue

```diff
@@ -29,29 +29,28 @@
 
 <script>
 export default {
-    data() {
-        return {
-                nav: [
-                    { content: 'Home', href: '/'},
-                    { content: 'Subjects', href: '/subject'},
-                    { content: 'Profile', href: '/profile'},
-                ],
-                info: [
-                    { content: 'Privacy Policy', href: '/'},
-                    { content: 'Terms of Use', href: '/'},
-                    { content: 'Help', href: '/'},
-                    { content: 'UNSW Handbook', href: '/'}
-                ],
-                social: [
-                    { content: 'facebook', href: '/'},
-                    { content: 'twitter', href: '/'}
-                ]
-        }
+  data() {
+    return {
+      nav: [
+        { content: 'Home', href: '/'},
```

Comment:
Pretty sure linter still complains that there's a space at start of { and not at end near }